spark/src/main/scala/geotrellis/spark/triangulation/StitchedDelaunay.scala

```diff
@@ -0,0 +1,171 @@
+package geotrellis.spark.triangulation
+
+import geotrellis.raster._
+import geotrellis.raster.triangulation.DelaunayRasterizer
+import geotrellis.spark.buffer.Direction
+import geotrellis.spark.buffer.Direction._
+import geotrellis.vector._
+import geotrellis.vector.io.wkt.WKT
+import geotrellis.vector.triangulation._
+
+import com.vividsolutions.jts.geom.Coordinate
+
+object StitchedDelaunay {
+
+  def directionToVertexOffset(d: Direction) = {
+    val increment = Int.MaxValue / 9
+    d match {
+      case TopLeft => 0
+      case Top => increment
+      case TopRight => 2 * increment
+      case Left => 3 * increment
+      case Center => 4 * increment
+      case Right => 5 * increment
+      case BottomLeft => 6 * increment
+      case Bottom => 7 * increment
+      case BottomRight => 8 * increment
+    }
+  }
+
+  def indexToVertex(neighbors: Map[Direction, (BoundaryDelaunay, Extent)]): Int => Coordinate =
+    { i =>
+      val increment = Int.MaxValue / 9
+      val group = i / increment
+      val index = i % increment
+      val dir = group match {
+        case 0 => TopLeft
+        case 1 => Top
+        case 2 => TopRight
+        case 3 => Left
+        case 4 => Center
+        case 5 => Right
+        case 6 => BottomLeft
+        case 7 => Bottom
+        case 8 => BottomRight
+      }
+      neighbors(dir)._1.pointSet.getCoordinate(index)
+    }
+
+  def indexToVertex(neighbors: Map[Direction, (DelaunayTriangulation, Extent)])(implicit dummy: DummyImplicit): Int => Coordinate =
```

Comment:
The `Direction` class should be duplicated in `geotrellis.raster.tiling.Direction` so we can move this class to `raster` package as well. The version of `Direction` in `spark` should be deprecated.